e2e-robot/Makefile

```diff
@@ -0,0 +1,43 @@
+
+.PHONY: setup dry-run with-filter with-tag all
+
+setup:
+	python3 -m venv .venv && \
+	source .venv/bin/activate && \
+	python3 -m pip install -r requirements.txt
+
+dry-run:
+	source .venv/bin/activate && \
+	robot \
```

Comment:
Same here, if you run `.venv/bin/robot` it should work without activating the environment.